### PR TITLE
Ms.rc8 changes

### PR DESCRIPTION
**Pull request by mariano54**
Blocks are re-generated

### CODE REVIEW

File: src/cmds/init.py [hunk 1]
```diff
@@ -377,9 +376,28 @@ def chia_init(root_path: Path):
...
+            return 0
+        else:
+            check_keys(root_path)
+            print(f"{root_path} already exists, no migration action taken")
+            return -1
+
+    print(f"Checking {os.path.expanduser(Path('~/.chia/testnet'))}")
+
+    # Migrate only from rc2 and up. Target addresses are not migrated.
     for version_number in range(5, 2, -1):
```
Comment (hoffmang9) [inline src/cmds/init.py:401]: Should we just drop this?

File: src/consensus/default_constants.py [hunk 2]
```diff
@@ -59,7 +59,7 @@
     "BLOCKS_CACHE_SIZE": 4608 + (128 * 4),
     "WEIGHT_PROOF_RECENT_BLOCKS": 1000,
     "MAX_BLOCK_COUNT_PER_REQUESTS": 32,  # Allow up to 32 blocks per request
-    "INITIAL_FREEZE_PERIOD": 5000,  # Transaction are disabled first 5000 blocks
+    "INITIAL_FREEZE_PERIOD": 42 * 4608,  # Transaction are disabled first 5000 blocks
```
Comment (hoffmang9) [inline src/consensus/default_constants.py:62]: I think we have to have a 5000 or 2500 block to test anything before we have the go/no go decision

**Review by hoffmang9 — CHANGES_REQUESTED** [review]

File: src/cmds/init.py (hunk 1 above)
Comment (mariano54) [reply src/cmds/init.py:401]: People are sitll migrating their plot directories and stuff, and many people are probably not farming and waiting for mainnet, so we want to migrate them too

File: src/consensus/default_constants.py (hunk 2 above)
Comment (mariano54) [reply src/consensus/default_constants.py:62]: Yes this is now being overridden in the overrides (config)

**Pull request merged**